components/zixflow/package.json

```diff
@@ -12,4 +12,4 @@
   "publishConfig": {
     "access": "public"
   }
-}
\ No newline at end of file
+}
```

Comment:
Package version should be updated and the `@pipedream/platform` should be installed